code/DebugBar.php

```diff
@@ -402,6 +403,11 @@ public static function renderDebugBar()
             $initialize = false;
         }
 
+        // Normally deprecation notices are output in a shutdown function, which runs well after debugbar has rendered.
+        // This ensures the deprecation notices which have been noted up to this point are logged out and collected by
+        // the MonologCollector.
+        Deprecation::outputNotices();
```

Comment:
With this we wouldn't need to require 4.13 in `composer.json`. It might be too prohibitive for adoption for folks who don't want to/can't update to CMS 4.13.
```suggestion
        // Normally deprecation notices are output in a shutdown function, which runs well after debugbar has rendered.
        // This ensures the deprecation notices which have been noted up to this point are logged out and collected by
        // the MonologCollector
        // This is available from CMS 4.13.
        if (method_exists(Deprecation::class, 'outputNotices')) {
            Deprecation::outputNotices();
        }
```

Reply:
This PR does nothing without the ability to output notices so anyone who doesn't want to upgrade wouldn't be affected anyway... but making this change breaks nothing so I'm okay with that.

Reply:
done